### ISSUE TITLE
Translate Blog Posts to French.

### ISSUE THREAD

**Issue by casanwugo** [rchain/bounties#889]
### Benefit to RChain

Translation of the RChain blog posts to other languages means good global marketing. Also, reach a larger audience and help expand the community building.

Translated contents will be posted both on the <a href="https://rchaincommunity.xyz/">RChain Worldwide Community Forum > French (Français) > Blog</a>, and <a href="https://medium.com/@annchidinma5">@Crypto Medium Blog</a>

### Original (English) Contents :

- <a href="https://blog.rchain.coop/start_coding_in_rholang/">Start coding in Rholang</a> (715 words)

- <a href="https://blog.rchain.coop/running-rnode-0-5-3-on-amazon-ec2/">Running RNode on Amazon AWS (EC2)</a> ( 641 words )

- <a href="https://blog.rchain.coop/reflective-ventures-deal-day/">Reflective Ventures Deal Day</a> ( 877 words )

- <a href="https://blog.rchain.coop/rchain-spotlight-hellofriend/">RChain Spotlight : Hellofriend</a> ( 443 words )

- <a href="https://blog.rchain.coop/running-rnodev-0-5-1-with-docker/">Running RNode (v 0.5.1) with Docker</a> ( 603 words )

- <a href="https://blog.rchain.coop/rchain-cooperative-rcon3/">RChain Cooperative Presents RCon3</a> ( 440 words )

- <a href="https://medium.com/rchain-cooperative/rchain-update-debrief-87-july-25th-2018-f5d7e8f3a53a">RChain Update (Debrief 87), July 25th, 2018</a> ( 1624 words )

- <a href="https://medium.com/rchain-cooperative/rchain-update-debrief-88-1st-august-2018-8058f8f94527">RChain Update (Debrief 88), 1st August, 2018</a> ( 1817 words )

### Budget and Objective

_Estimated Budget of Task:_ **$[ 1260.16 ]**  [ 1145.6 + 114.56 (For Markdown Conversion) ]
_Estimated Timeline Required to Complete the Task:_ **[3 weeks]**
_How will we measure completion?_ **[ Fill the *Translated (French) Contents* section above ]**

### Translation Checklist

*for non translation issues, delete this section*

  - Why translate this document?
    - It's important to translate because ____
    - [x] It's an RChain blog item on [Medium](https://medium.com/rchain-cooperative)
    - [x] It's an RChain blog item on https://blog.rchain.coop/
    - [ ] It's a document on https://developer.rchain.coop/ that has been endorsed
          for translation by @Jake-Gillberg, @MParlikar, or @KellyAtPyrofex 
  - How can people who do not speak this language be assured of the quality of the translation? (all are required)
    - [x] several reviewers are involved (_assign the ticket to them_)
    - [x] at least two of them have high proficiency in _the target language_
    - [x] at least two of them have high proficiency in English
    - [x] at least two of them have either
       - [x] formal training: degree / certificate: ___ from institution: _____
       - [x] a track record of quality work on rchain bounties: #711, #808, #846 ...
       - [ ] another body of work readily available for community review: _____
  - What about maintenance? (choose one)
    - [x] the source document is stable; further translation work is not expected
    - [ ] our plan to keep the translation up to date is: ____

_**Word counting and pricing for translations will be done with [The World's Fastest Human Translation Service](https://customer.stepes.com/instant-translation-quote/#)**_

**Comment by AyAyRon-P** [comment]
@casanwugo I'm not seeing $3k value in these translations. I would advise you to take this as an opportunity to write original commentary on each of these blogs, debriefs, and articles. If you were to do this there would be substantially more value and and this issue could be worth the $3023.56 bounty you're claiming. 

@pavlos1851 @zsluedem

**Comment by ddayan** [comment]
I don’t think there’s a need for these translations

**Comment by philipandri002** [comment]
@AyAyRon-P sorry how do mean by "I would advise you to take this as an opportunity to write original commentary on each of these blogs, debriefs, and articles". 

Pls help with an example of what you mean or a pointer to other translators to be sure what exactly you would want the content to look like.

Thanks

**Comment by casanwugo** [comment]
@AyAyRon-P... Thanks for reaching out, and also for your advise. But as @philipandri002 stated,,.. Could you please point to me what kind of commentary you mean so i could include those in my work?  

> If you were to do this there would be substantially more value 
I'm all about adding value to my translations.

**Comment by AyAyRon-P** [comment]
@casanwugo what we are looking for in the "value added" commentary is your summary of the article. Highlight what was discussed, talk about any action items from the article, breakdown any software releases, summarize any technology in layman's terms for greater audience reach, and in some cases your own opinion and take on what the article means to you. At the end you could add a line saying "Notes from the Editor" or something to that affect so readers know that it's not part of the original article/blog/debrief. Does this make sense?

**Comment by casanwugo** [comment]
Yes @AyAyRon-P,,, i have done that already. Thanks alot!

**Comment by casanwugo** [comment]
The part "Comment pourrions-nous faciliter la tâche à un propriétaire de bar de donner au hasard une bière à ses clients toutes les heures de manière automatisée, sécurisée et vérifiable publiquement?" ("How could we make it easy for a bar owner to randomly give away a beer to their customers every hour in an automated, secure, and publicly verifiable way?") is from the original text.  

While the part below is a coinage of my comment, written with the intention of making the above paragragh catchy. Thereby, attracting the readers attention to the whole story.

Vous demandez peut-être "Bière en blockchain?" Oui la bière! Jetons de fidélité, clients et propriétaires d’entreprise. Découvrez-en plus dans cette histoire intrigante ... (You might be asking "Beer in blockchain?" Yes beer! Loyalty tokens, customers, and business owners.. Find out more in this intriguing story...)
 
What are your thoughts?

**Comment by AyAyRon-P** [comment]
@casanwugo this is far from the editor's notes I would expect. That's 2 lines of text and not commentary in the slightest bit, until this gets resolved I'm putting in a reduced bounty on this. 

I also feel that the weekly debriefs are not nearly as valuable as actual blogs or articles, this goes for all languages not just the ones you are doing. 

CC @pavlos1851 @zsluedem

**Comment by AyAyRon-P** [comment]
Here are examples of an EDITOR'S NOTE: 

https://harpers.org/blog/2018/08/inside-the-september-2018/

http://discovermagazine.com/2013/september/01-editors-note

https://www.nytimes.com/2004/02/15/nyregion/editors-note-editors-note-402559.html

**Comment by luigidemeo** [comment]
we do not need more translations unless the original content was widely consumed. Please post metrics as to why the content needs to be translated.
